### PR TITLE
Removing texts part c#161

### PR DESCRIPTION
**Pull request by amabelleS**
# Description
Removed from: FilterPositionsByStartTimeSelector, GapsPage, DisplayGapsPercentage, DateSelector, and TimelinePage.

### CODE REVIEW

File: src/locale/en.json [hunk 1]
```diff
@@ -94,5 +94,7 @@
   "bug_type_message":"Please enter the type of request",
   "bug_type_bug":"Bug",
   "bug_type_feature":"Feature Request",
-  "funding_paragraph": "The Databus project is developed by the Hasadna for public knowledge, by the contributions of volunteers, and based upon"
+  "funding_paragraph": "The Databus project is developed by the Hasadna for public knowledge, by the contributions of volunteers, and based upon",
```
Comment (NoamGaash) [inline src/locale/en.json:97]: Space removed by purpose?

File: src/pages/gaps/index.tsx [hunk 2]
```diff
@@ -64,6 +39,32 @@ const GapsPage = () => {
   const [gapsIsLoading, setGapsIsLoading] = useState(false)
   const [onlyGapped, setOnlyGapped] = useSessionStorage('onlyGapped', false)
 
+  function formatTime(time: Moment) {
```
Comment (NoamGaash) [inline src/pages/gaps/index.tsx:42]: Why did you move it inside the component?

**Review by NoamGaash — APPROVED** [review]

File: src/locale/en.json (hunk 1 above)
Comment (amabelleS) [reply src/locale/en.json:97]: > Space removed by purpose?

No. Maybe Lint complained, but I can't even see what space we are talking about... just that there is a change in "funding_paragraph" that I did not mean to make.

File: src/pages/gaps/index.tsx (hunk 2 above)
Comment (amabelleS) [reply src/pages/gaps/index.tsx:42]: > Why did you move it inside the component?

I feel ashamed about why, and I am trying to think of another solution.  
You can call t / ``` const { t } = useTranslation() ``` outside of a react component because it is a hook. So I moved the functions that use t inside of the react component function. Very dirty, I need a shower now:)

Comment (NoamGaash) [reply src/pages/gaps/index.tsx:42]: Oh, you're right. Its not dirty - I just didn't expect moving the time format to the translation file
